crates/aptos-rosetta/src/account.rs

```diff
@@ -136,55 +136,53 @@ async fn get_balances(
                     }
 
                     let store: Store = bcs::from_bytes(&bytes)?;
-                    if account.is_total_stake() {
-                        // For total stake, collect all underlying staking contracts and combine
-                        let mut total_stake: Option<u64> = None;
-                        maybe_operators = Some(vec![]);
-                        for (operator, contract) in store.staking_contracts {
-                            // Keep track of operators
-                            maybe_operators.as_mut().unwrap().push(operator);
-                            match get_total_stake(
+                    let mut total_requested_balance: Option<u64> = None;
+                    maybe_operators = Some(vec![]);
+                    for (operator, contract) in store.staking_contracts {
+                        // Keep track of operators
+                        maybe_operators.as_mut().unwrap().push(operator);
+                        match get_stake_balances(
+                            rest_client,
+                            &account,
+                            contract.pool_address,
+                            version,
+                        )
+                        .await
+                        {
+                            Ok(Some(balance)) => {
+                                total_requested_balance = Some(
+                                    total_requested_balance.unwrap_or_default()
+                                        + u64::from_str(&balance.value).unwrap_or_default(),
```

Comment:
This seems complex for what it is, but I wrote it so I guess maybe there was a reason 🤷🏻‍♂️